lib/lotus/root.rb

```diff
@@ -0,0 +1,7 @@
+require 'pathname'
+
+module Lotus
+  def self.root
+    Lotus::Configuration.new.root
```

Comment:
Then you have lotus container, https://github.com/lotus/lotus/blob/master/lib/lotus/container.rb inside you have access to configuration